frontend/model/state.js

```diff
@@ -510,6 +525,44 @@ const getters = {
       return 1
     }
     return 7
+  },
+  currentChatRoomId (state, getters) {
+    return state.currentChatRoomIDs[state.currentGroupId] || null
+  },
+  isPublicChatRoom (state, getters) {
+    return (chatRoomId: string) => {
+      return state[chatRoomId]?.attributes.privacyLevel !== CHATROOM_PRIVACY_LEVEL.PRIVATE
+    }
+  },
+  isJoinedChatRoom (state, getters) {
+    return (chatRoomId: string, username?: string) => {
+      username = username || state.loggedIn.username
+      return !!state[chatRoomId]?.users && !!state[chatRoomId]?.users[username]
```

Comment:
It seems to be possible to write this like so:

```js
return !!state[chatRoomId]?.users?.[username]
```

Reply:
Yeap. Nice catch.